spec/app_store/receipt_collection_spec.rb

```diff
@@ -26,7 +28,7 @@
 
     it 'has positive overdue days' do
       overdue = subject.overdue_days
-      overdue.must_be_instance_of Fixnum
+      overdue.is_a?(Integer).must_be_true
```

Comment:
maybe we shouldn't care this much about types anyway. Maybe asserting that it is `.positive?` should suffice?